### PR TITLE
Enhancement/90/mobile support

### PR DESCRIPTION
**Pull request by xcaverxiii**
Related Issue #90

Fixed some mobile UX issues.

chanded title css
changed navbar alignment
changed slider images per slide from 8 to 1 for mobile
changed artwork related tabs size for each tab to 100%

### CODE REVIEW

**Comment by jschurse** [comment]
I've got some real bad performance issues, if the slider only shows one element.
It takes 12seconds for the page to fully load (compared to 1.2 seconds with 8 Slider elements). The navigation between two artworks takes an equal time.
I've tracked the problem to the zone.js library which angular uses for the change detection cyle, but I'm not quite sure how (and why) this change causes such issues.

**Comment by xcaverxiii** [comment]
> I've got some real bad performance issues, if the slider only shows one element.
> It takes 12seconds for the page to fully load (compared to 1.2 seconds with 8 Slider elements). The navigation between two artworks takes an equal time.
> I've tracked the problem to the zone.js library which angular uses for the change detection cyle, but I'm not quite sure how (and why) this change causes such issues.

I can not reproduce your problem. Maybe its located at your local maschine.
There ist just a need of time to load all the images.

**Comment by jschurse** [comment]
I checked on different physical systems (but similar software environments) in different browsers.
To better illustrate the problem I'm having I recoded a short video:
https://gfycat.com/unconscioushealthycrayfish
What you can't hear in the video is the CPU Cooler running at 100% while loading the content.

@JamieArmstrongArndt  @mxxbe  @marvinseith Can anybody else verify this problem?

**Comment by marvinseith** [comment]
It works perfectly on my machine, can't reproduce the high CPU usage @jschurse. Maybe you have some corrupt packages installed? We should take a look together tomorrow.

**Comment by marvinseith** [comment]
After the last commit, the number of displayed pictures on mobile as lowered, which got the performance back to normal.

**Comment by xcaverxiii** [comment]
Performance bug is fixed by only show 20 related artworks when is mobile

File: app/src/app/shared/components/slider/slider.component.ts [hunk 1]
```diff
@@ -52,29 +53,37 @@ export class SliderComponent implements OnChanges {
...
+    this.checkIsMobile();
+  }
 
   /** Hook that is called when any data-bound property of a directive changes. */
   ngOnChanges(changes: SimpleChanges) {
     // rebuild slides if slider items input changed.
     if (changes.items && this.items) {
-      this.buildSlides();
+      // Slice items to max 20 related artworks when mobile
+      this.slides = this.buildSlides(this.isMobile ? this.items.slice(0, 20) : this.items);
```
Comment (jschurse) [inline app/src/app/shared/components/slider/slider.component.ts]: Maybe we should extract a variable/constant for this magic number.

**Pull request merged**